src/Background_Test.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+
+namespace Tribe\Extensions\Promoter;
+
+
+use Tribe\Extensions\Promoter\Health\Health_Exception;
+use Tribe\Extensions\Promoter\Health\Tests\Connection;
+use Tribe\Extensions\Promoter\Health\Tests\Endpoints;
+use Tribe\Extensions\Promoter\Health\Tests\ET;
+use Tribe\Extensions\Promoter\Health\Tests\Functions_Requirements;
+use Tribe\Extensions\Promoter\Health\Tests\License_Presence;
+use Tribe\Extensions\Promoter\Health\Tests\Runnable;
+use Tribe\Extensions\Promoter\Health\Tests\TEC;
+use Tribe\Extensions\Promoter\Health\Tests\Token_Presence;
+
+class Background_Test {
+	public static $tests = [
+		TEC::class,
+		ET::class,
+		Endpoints::class,
+		Functions_Requirements::class,
+		License_Presence::class,
+		Token_Presence::class,
+		Connection::class,
+	];
+
+	public static function run_test() {
+		check_ajax_referer( 'health-check-site-status' );
+
+		$result = [
+			'label'       => 'Promoter - Site Health Review',
+			'completed'   => true,
+			'status'      => 'good',
+			'badge'       => [
+				'label' => 'Promoter',
+				'color' => 'blue',
+			],
+			'description' => 'Everything looks correctly in your WordPress installation.',
```

Comment:
remove "ly" from "correctly". should it be translatable?

Reply:
Thanks, in terms of translatable strings, as this first version is mostly for support, I think we can update the strings and make them "translatable" for the next version instead.

Reply:
That's not how we normally do Extensions and these are user-facing messages so I'd recommend making them translatable and following the pot generation command from https://github.com/mt-support/tribe-ext-extension-template/blob/master/README.md

Reply:
Thanks I did some updates and added the required files. Let me know if you have additional feedback before I can move forward with the release of this extension.